deploy/cloudbuild-release.yaml

```diff
@@ -0,0 +1,6 @@
+# This cloudbuild is run on the creation of new tags, which should signify releases.
+steps:
+  - name: "gcr.io/cloud-builders/docker"
+    args: ["build", "-f", "deploy/Dockerfile",
+           "-t", "gcr.io/kaniko-project/kritis-server:$TAG_NAME", "."]
+images: ["gcr.io/kaniko-project/kritis-server:$TAG_NAME"]
```

Comment:
This should be `kritis-int-test`

Reply:
Actually no, We should be releasing the images to kritis-project. I will swtich these to kritis-project.